OrchardCore.Commerce.Tests/OrchardCore.Commerce.Tests.csproj

```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.TestPlatform.TestHost" Version="16.7.1" />
-    <PackageReference Include="OrchardCore.ContentFields" Version="1.0.0-rc2-14744" />
+    <PackageReference Include="OrchardCore.ContentFields" Version="1.0.0-rc2-*" />
```

Comment:
I really, really dislike using a star here given the breaking changes (such as new methods on interfaces) that Orchard Core introduces routinely. I think we should be deliberate in upgrading the version we use, so people can clone, restore and build out of the box.

Reply:
I agree with this, changed back to fixed version.